### ISSUE TITLE
`ktlint_standard_function-expression-body = disabled` not honored

### ISSUE THREAD

**Issue by swankjesse** [pinterest/ktlint#2867]
## Expected Behavior
Given `.editorconfig`:
```
ktlint_standard_function-expression-body = disabled
```

And this source file, `hello.kt`
```kotlin
fun hello(): String {
  return "hello"
}
```

Applying ktlint should not apply the expression body transformation to the `hello()` function.

## Observed Behavior

```kotlin
fun hello(): String = "hello"
```

## Steps to Reproduce

I’m using `com.pinterest.ktlint:ktlint-cli:1.4.1` via Spotless. (I assume Spotless isn’t to blame here.)

**Comment by paul-dingemans** [comment]
This is not a problem caused by Ktlint. I expect that Spotless provides an `editorConfigOverride` to the `KtLintRuleEngine` that actually overrides what you have specified in your `.editorconfig`.

**Comment by paul-dingemans** [comment]
Please address issue with Spotless team, and add a link to their issue if possible. Closing the issue for now.
